MTM-51375: Update the What's New section about the mass session revocation (#155)
SoftwareAG/c8y-releasenotes: content/release-10-17-0/whatsnew-10-17-0.md

Cumulocity IoT supports new standard OAuth2 access tokens with the "at+jwt" type header parameter. For more information on this parameter, see [OAuth2 Access Tokens](https://datatracker.ietf.org/doc/html/rfc9068#name-header).
Single-sign-on users managed by new authorization servers (that is, generating OAuth2 access tokens with type "at+jwt") can thus login successfully into Cumulocity IoT.

> ### User menu

Reviewer: ```suggestion
### Authentication
```

> #### Access token revocation for all tenant users

Reviewer 2: Replace Access token with Session tokens

> The user menu of admin users contains a new button for mass token revocation. The new button can be used to revoke all user tokens, used by OAI-Secure or Single-Sign-On to access the platform.

Reviewer: ```suggestion
The user menu of admin users contains a new option for revoking tokens. This option can be used to revoke all user tokens, which are used by OAI-Secure or Single-Sign-On for access to the platform, in one step.
```

> The user menu of admin users contains a new option for revoking tokens. This option can be used to revoke all user tokens, which are used by OAI-Secure or Single-Sign-On for access to the platform, in one step.

Reviewer 2: Let us replace the option to be available for Tenant Administrator in the users page.

Reviewer 2: Let us call the Session tokens, not the user tokens.

This feature is useful, for example, if there is a security issue with the tokens currently in use. Revoking the tokens requires the re-login of all users.

> ![User token revocation](/images/release-notes/admin-token-revocation.png)

Reviewer 2: Replace the screenshot with the one from Users page.
